modifications/kevin_nickels/README.md

```diff
@@ -12,20 +20,26 @@ The [LX-16A](https://www.amazon.com/Hiwonder-LX-16A-Robotic-Controller-Control/d
 
 Since I've had problems with threads, and even threaded inserts, pulling out of plastic before, I also generated an alternative bracket in the same OnShape CAD, Bracked w Captured Nuts.  This is designed to be used with [M3x40mm screws](https://www.amazon.com/BNUOK-120pcs-Socket-Threads-Spanner/dp/B0DJQFXKHG/).
 
-![CAD rendering of LX-224 servo bracket featuring captured nuts for enhanced strength, showing the bracket attached to a steering knuckle within a mechanical assembly.](LX-224 w captured nuts_cad.png)
+![CAD rendering of LX-224 servo bracket featuring captured nuts for enhanced strength, showing the bracket attached to a steering knuckle within a mechanical assembly.](LX-224_w_captured_nuts_cad.jpg)
 
 ## M3 SetScrews
 As was mentioned in the Discussion section of [CJ - a sawppy variant](https://hackaday.io/project/165094-cj-a-sawppy-variant) on Hackaday, some of us have had issues with heat-set inserts pulling out.  As shown in this CAD and the pictures below, I ended up printing a Nut Carriage to get good strength and print resolution in the XY direction, then printing the knuckle in the standard orientation, so that the M3 nut won't spin when tightened against the flat in the 8mm shaft.  Then, I used an M3x16 screw as a setscrew instead of the tiny ones called out in the original.  We'll see how this holds up over time!
 
 ![CAD of steering knuckle with nut insert](Knuckle_CAD.jpg)
 ![image of steering knuckle with nut insert](knuckle_w_nuts.jpg)
+![image of steering knuckle with nut insert installed](knuckle_w_setscrew.jpg)
  
 
 ## Joystick Controller Modifications
-I also modified the [Handheld Arduino Joystick controller](https://cad.onshape.com/documents/47ee84411a76e86eae57ea0f/w/f70ae835a2d548c4b95fa46e/e/c8538a52492e953cdcdd3db5?renderMode=0&uiState=6887d5ec43a024454e4cecf6) to accept an [RJ9](https://www.ebay.com/itm/317094616170) so that I can connect my controller to my rover with a standard coiled phone cord.
+I also modified the [Handheld Arduino Joystick controller](https://cad.onshape.com/documents/47ee84411a76e86eae57ea0f/w/f70ae835a2d548c4b95fa46e/e/c8538a52492e953cdcdd3db5?renderMode=0&uiState=688a32537f05d177d7ca5baf) to accept an [RJ9](https://www.ebay.com/itm/317094616170) so that I can connect my controller to my rover with a standard coiled phone cord.
```

Comment:
I've been fiddling with these OnShape links and my current hypothesis is that the suffix "?renderMode=0&uiState=[...]" is somehow specific to your browser. If I use the full URL I get an error but if I remove that suffix I could access your OnShape CAD.

Please truncate URL by removing "?" and everything after that. (I arbitrarily chose this OnShape URL to comment upon but the request applies to all of them on this page.)

Reply:
I don't think that I had "Anyone with link" set to "Can View" at first - I assumed that when you copied a link using the chain icon at the top that it set the permissions properly.  

I just tested all four of the OnShape links in an incognito window and in a chrome, edge, and firefox windows with no OnShape cookie, and they all worked fine for me.  Would you mind checking again?  BTW, what browser are you using?

Reply:
Same trio: WIndows Chrome, Firefox, and Edge. But it worked today so maybe it was a temporary glitch yesterday on OnShape's end. Shrug. Looks good to go now.

Reply:
... and it's giving me 403 Forbidden again. I don't know what variables changed between the earlier attempt and my most recent attempt. I can't very well demand you to fix something I can't articulate. And since it does work occasionally (if for reason's I don't understand) I'll to merge this now.

![Trippy Onshape CAD 403 Forbidden](https://github.com/user-attachments/assets/62889fd4-c4f8-44f8-a51d-e98c25c177ba)